fdbclient/include/fdbclient/MetaclusterManagement.actor.h

```diff
@@ -1544,6 +1544,28 @@ Future<Void> deleteTenant(Reference<DB> db, int64_t id) {
 	return Void();
 }
 
+template <class Transaction>
+Future<std::vector<std::pair<TenantName, int64_t>>> listTenantIdsTransaction(Transaction tr,
+                                                                             TenantName begin,
+                                                                             TenantName end,
+                                                                             int limit) {
+	tr->setOption(FDBTransactionOptions::READ_SYSTEM_KEYS);
```

Comment:
We shouldn't put this option into the transaction versions because that could escalate the privileges of the caller. Instead, the contract is that the caller sets the appropriate options.